cmd/kubelet/app/options/options.go

```diff
@@ -193,6 +193,10 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 		return fmt.Errorf("unsupported CRI runtime: %q, only %q is currently supported", f.ContainerRuntime, kubetypes.RemoteContainerRuntime)
 	}
 
+	if f.RemoteRuntimeEndpoint == "" {
```

Comment:
note: maybe we even test for it being a valid socket address as an additional improvement. The only problem with it will be that some setups may not specify `unix://` prefix. So this check is OK for back compat